src/cli/share-manager.test.ts

```
@@ -0,0 +1,201 @@
+
+  it('copy mode forks: mutating the copy does not touch the source', () => {
+    const source = path.join(anchorDir, 'skills');
+    const target = path.join(accountDir, 'skills');
+    write(path.join(source, 'my-skill', 'SKILL.md'), 'v1');
+
+    shareItem(source, target, 'copy');
+    expect(getShareState(target).kind).toBe('own');
+    fs.writeFileSync(path.join(target, 'my-skill', 'SKILL.md'), 'forked');
+    expect(fs.readFileSync(path.join(source, 'my-skill', 'SKILL.md'), 'utf-8')).toBe(
+      'v1',
+    );
```

ESLint

[error] Replace `fs.readFileSync(path.join(source,·'my-skill',·'SKILL.md'),·'utf-8')).toBe(⏎······'v1',⏎····` with `⏎······fs.readFileSync(path.join(source,·'my-skill',·'SKILL.md'),·'utf-8'),⏎····).toBe('v1'`

(prettier/prettier)